src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -63,6 +82,10 @@ export default Utils.withCancellationSignal(class LaunchAnalysisModal extends Co
       ]),
       div({ style: { color: colors.danger(), overflowWrap: 'break-word' } }, [
         h(Fragment, wrappableOnPeriods(launchError))
+      ]),
+      location && locationType && span({}, [
+        'Data produced by this run will be stored in the following workspace bucket: ',
```

Comment:
Users can specify custom output locations, right?

The following is more concise and possibly more accurate by not over-promising:
```suggestion
        'Workspace bucket:',
```

Reply:
I didn't think that they could? At least not in Terra?

Reply:
(at least, at least not via this submit dialog?)

Reply:
I could write a task that uploads a localized file to an arbitrary bucket, I'm pretty sure I've heard of people doing this or similar somewhere

It's not likely to happen in Terra, OTOH making statements about where data always goes is a commitment